docs/content/users/usage/commands.md

```diff
@@ -971,15 +971,33 @@ ddev npm update
 
 ## `nvm`
 
-Run [`nvm`](https://github.com/nvm-sh/nvm#usage) inside the web container (global shell web container command). (Use of `ddev nvm` is discouraged because `nodejs_version` is much easier to use, can specify any version, and is more robust than using `nvm`.)
+Run [`nvm`](https://github.com/nvm-sh/nvm#usage) inside the web container (global shell web container command).
+
+!!!tip
+    Use of `ddev nvm` is discouraged because `nodejs_version` is much easier to use, can specify any version, and is more robust than using `nvm`.
```

Comment:
> Use of `ddev nvm` is discouraged because it doesn't affect subsequent `ddev` commands.

No, this is not the case. The only thing that doesn't work properly is `ddev nvm use <version>`, and I added a warning about that in this PR.

Reply:
Ahh, okay, I must have misread that then.